.circleci/config.yml

```diff
@@ -1645,6 +1652,22 @@ workflows:
             parameters:
               mips_word_size: [32, 64]
 
+  scheduled-forge-coverage:
+    triggers:
+      - schedule:
+          cron: "0 0,4,8,12,16,20 * * *"
+          filters:
+            branches:
+              only:
+                - develop
+    jobs:
+      - contracts-bedrock-build:
+          name: contracts-bedrock-build-coverage
+          profile: cicoverage
+      - contracts-bedrock-coverage:
```

Comment:
You should be able to run this directly without needing `contracts-bedrock-build-coverage`